lib/githooks/runner.rb

```diff
@@ -98,6 +98,9 @@ def attach
       gitrunner ||= SystemUtils.which('githooks-runner')
       gitrunner ||= (GitHooks::BIN_PATH + 'githooks-runner').realpath
 
+      # When repos are cloned, sometimes the hooks directory isn't created
+      FileUtils.mkdir(@repository.hooks) unless File.directory?(@repository.hooks)
```

Comment:
`@repository.hooks` is actually a Pathname object - no need to call FileUtils.mkdir or File.directory? on it. You can simplify by doing:

```
# the guard clause isn't really necessary as #mkpath actually handles that for you. However,
# sometimes verbosity adds clarity. I leave it to you to decide the best approach.
@repository.hooks.mkpath unless @repository.hooks.directory?
```